ynput/ayon-core
# Houdini: Add generic filepath loader

## Changelog Description

Add generic filepath loader that can be loaded and managed via AYON.

## Additional info

This is useful if for a particular workflow there is no existing loader yet. A Houdini artists can load as the generic filepath loader and then reference the relevant Houdini parm to use the exact value. The benefit is that this filepath will be managed and can be updated as usual.

## Testing notes:

Each of the below should work:

1. Load some paths via loader
2. Update / switch folders via scene inventory (manage)
3. Remove via scene inventory  (manage)
